Emma/emma_libs/memoryManager.py

```diff
@@ -389,6 +407,12 @@ def _createTeamScalePath(memEntryRow):
                         break
                     else:
                         sc().wwarning("The input is not a valid dec number. Please enter the start and end address again: \n")
-                createSvgReport(startRegion, endRegion)
+                print("Enter the scaling value. If you don't enter the valid float number, scaling value 1 will be assumed.")
```

Comment:
```suggestion
                print("Enter a scaling value (float). If empty a scaling value `1` will be assumed.")
```

We should promt again if the scaling value is invalid.

Also: does it have to be comma or dot separated? -> `1.0` or `1,0`
What happens if the wrong one was entered?